packages/manager/modules/pci/src/projects/project/workflow/add/resources/resources.controller.js

```diff
@@ -1,6 +1,7 @@
 import get from 'lodash/get';
 
 import Instance from '../../../../../components/project/instance/instance.class';
+import Flavors from '../../../../../components/project/flavors-list/flavor.class';
```

Comment:
Could be named `Flavor` instead of `Flavors`?

Reply:
I am not sure to understand the question:
-> if you mean to rename the class, i should change all Flavors occurrences into the whole project
-> if you mean to change imported Flavors to Flavor, i can use an alias
Which point are you talking about?

Reply:
I think what @cbourgois meant is to rename your import to use singular as the model represent a Flavor and not several one